### DISCUSSION TITLE
Fail2Ban

### DISCUSSION THREAD

**Discussion by ghost** [nextcloud/all-in-one#361] [Ideas] [answered]
Intigrating Fail2Ban.?

**Comment by szaimen** [comment]
This may be implemented sometime in the future but is for the stable release out of scope.
For now we rely on Nextclouds own bruteforce protection.

Probably easiest to build it on our own. Here are some things that might be needed for the container to work correctly: https://github.com/crazy-max/docker-fail2ban#command-line

**Comment by szaimen** [comment, marked as answer]
We will add documentatio on this with https://github.com/nextcloud/all-in-one/pull/851

**Reply by Aceriz** [reply]
> We will add documentatio on this with #851

I have looked at your instructions on the main page.  

I am wondering if you perhaps know within an unraid setup where I would be able to find the "/var/lib/docker/volumes/nextcloud_aio_nextcloud/_data/data/nextcloud.log "     

on unraid the docker is setup as a .img   file at 
/mnt/user/system/docker/docker.img   so not sure how to direct towards the above location


 thanks for help

**Reply by szaimen** [reply]
Honestly I don't know where the files are stored on unraid. You may be able to find it out with `sudo docker inspect nextcloud_aio_nextcloud`

**Reply by Aceriz** [reply]
> Honestly I don't know where the files are stored on unraid. You may be able to find it out with `sudo docker inspect nextcloud_aio_nextcloud`

that worked for me to identify the file location thanks.  I will now just work on the other part of fail2ban
